General info and CNCF Stories (#2)
cloud-native-eastern-canada/meetups: README.md

# Cloud Native Eastern Canada Meetups

Github repository with all Meetup information and slides from Cloud Native Eastern Canada Meetups:
* [Online Canadian Cloud Native Stories](stories/README.md)
* [Online Canadian Tech Talks](techtalks/README.md)
* [Montreal](montreal/README.md)
  * Julia Simons Director of Marketing, [CloudOps](cloudops.com)
  * Ayrat Khayretdinov [@archyufa](https://github.com/archyufa), [CNCF Ambassador](https://www.cncf.io/people/ambassadors/)
  * Anthony Dahane [@anthonydahanne](https://github.com/anthonydahanne), Software engineer, [Adaptive](https://weareadaptive.com/)

>   * Alex Gervais [@alex_gervais](https://twitter.com/alex_gervais), Senior Software Engineer, [Datawire](Datawire.io)

Reviewer: ```suggestion
  * Alex Gervais [@alex_gervais](https://twitter.com/alex_gervais), Senior Software Engineer, [Datawire](https://www.datawire.io/)
```

* [Ottawa](ottawa/README.md)
  * Chris Carty, [@macintoshprime](https://twitter.com/macintoshprime) Customer Engineer, [Google Cloud](https://cloud.google.com/) 
  * Suvro Ghosh,  Software engineer, [Ribbon](https://ribboncommunications.com/)
  * Raymond Maika, Engineering Team Lead at [CENGN](https://www.cengn.ca/)
* [Quebec](quebec/README.md)
  * Sebastien Thomas [@prune998](https://github.com/prune998) SRE, [Tetrate](https://www.tetrate.io/)
  * Carol Tardif, [Ticketmaster](https://www.ticketmaster.ca/)
* [Toronto](toronto/README.md)

>   * Jason Hadi [jasonhadi](https://twitter.com/jasonhadi), SRE, [Ecobee](https://www.ecobee.com/en-us/)

Reviewer 2: ```suggestion
  * Jason Hadi [@jasonhadi](https://github.com/jasonhadi), SRE, [ecobee](https://www.ecobee.com/)
```

  * Dima Kassab, [@dima_kassab](https://twitter.com/dima_kassab), Customer Engineer, [Google Cloud](https://cloud.google.com/) 
* [Waterloo-Kitchener](waterloo-kitchener/README.md)  
  * Chris Volik [psycotica0](https://github.com/psycotica0), Production engineer, [Shopify](https://www.shopify.ca/)

## Join our Community!

### Slack

> Join our vibrant K8s Canada Slack community to ask you questions, collaborate bettween other Canadian cities, meet people with same interest and learn about latest CNCF, Kubecon, KCDs news - [http://K8scanadaslack.herokuapp.com](http://K8scanadaslack.herokuapp.com)

Reviewer 2: ```suggestion
Join our vibrant K8s Canada Slack community to ask your questions, collaborate between other Canadian cities, meet people with same interests, and learn about the latest CNCF, Kubecon, and KCDs news - [http://K8scanadaslack.herokuapp.com](http://K8scanadaslack.herokuapp.com)
```
